qa/lock/lockables_test.go

```diff
@@ -0,0 +1,77 @@
+package lock
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/qa/lock/internal"
+)
+
+func TestGenerateBlobName(t *testing.T) {
```

Comment:
This test looks like it should live in `internal` as well (only uses `GenerateBlobName`).